site/about.html

```diff
@@ -1,28 +1,42 @@
+<!doctype html>
+
 <html>
 	<head>
 		<title>About me</title>
-		<link rel="stylesheet" href="style.css" >
+
+		<link rel="stylesheet" href="style.css">
+		<link href='https://fonts.googleapis.com/css?family=Oswald:700' rel='stylesheet' type='text/css'>
+		<link href='https://fonts.googleapis.com/css?family=Lato:400,900' rel='stylesheet' type='text/css'>
 	</head>
 
 	<body>
-		<nav>
-			<a href="about.html">About Me</a>
-			<a href="portfolio.html">Portfolio</a>
-		</nav>
+		<ul class="top_nav">
+			<li class="top_nav_item"><a href="about.html" class="active">About me</a></li>
```

Comment:
Great use of nested tags.  For next time, start each tag on a new (indented) line.  It will make your HTML much easier to read (especially when your pages get more complicated).